konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/format/JavaNDArrays.java

```diff
@@ -88,5 +88,591 @@ public Float5Array(float[][][][][] array) {
+        }
+
+        @Override
+        public long[] shape() {
+            return shape;
+        }
+    }
+
+    public static class Int321Array extends BaseInt32Array<int[]>{
+        public Int321Array(int[] array) {
+            super(array, new long[]{array.length});
+        }
+    }
+
+    public static class Int322Array extends BaseInt32Array<int[][]>{
+        public Int322Array(int[][] array) {
+            super(array, new long[]{array.length, array[0].length});
+        }
+    }
+
+    public static class Int323Array extends BaseInt32Array<int[][][]>{
+        public Int323Array(int[][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length});
+        }
+    }
+
+    public static class Int324Array extends BaseInt32Array<int[][][][]>{
+        public Int324Array(int[][][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length, array[0][0][0].length});
+        }
+    }
+
+    public static class Int325Array extends BaseInt32Array<int[][][][][]>{
+        public Int325Array(int[][][][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length, array[0][0][0].length, array[0][0][0][0].length});
+
+        }
+    }
+
+    private static abstract class BaseInt64Array<T> extends BaseNDArray<T>{
+        protected final long[] shape;
+        public BaseInt64Array(T array, long[] shape) {
+            super(array);
+            this.shape = shape;
+        }
+
+        @Override
+        public NDArrayType type() {
+            return NDArrayType.INT64;
+        }
+
+        @Override
+        public long[] shape() {
+            return shape;
+        }
+    }
+
+    public static class Int641Array extends BaseInt64Array<long[]>{
+        public Int641Array(long[] array) {
+            super(array, new long[]{array.length});
+        }
+    }
+
+    public static class Int642Array extends BaseInt64Array<long[][]>{
+        public Int642Array(long[][] array) {
+            super(array, new long[]{array.length, array[0].length});
+        }
+    }
+
+    public static class Int643Array extends BaseInt64Array<long[][][]>{
+        public Int643Array(long[][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length});
+        }
+    }
+
+    public static class Int644Array extends BaseInt64Array<long[][][][]>{
+        public Int644Array(long[][][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length, array[0][0][0].length});
+        }
+    }
+
+    public static class Int645Array extends BaseInt64Array<long[][][][][]>{
+        public Int645Array(long[][][][][] array) {
+            super(array, new long[]{array.length, array[0].length, array[0][0].length, array[0][0][0].length, array[0][0][0][0].length});
+
+        }
+    }
+
+    private static abstract class BaseUint8Array<T> extends BaseNDArray<T>{
```

Comment:
Unsigned integer types also aren't supported in Java, so let's remove all the UInt classes here too